ydb/core/kqp/ut/tx/kqp_locks_tricky_ut.cpp

```diff
@@ -324,6 +326,7 @@ Y_UNIT_TEST_SUITE(KqpLocksTricky) {
 
             auto result = runtime.WaitFuture(future);
             UNIT_ASSERT_VALUES_EQUAL_C(result.GetStatus(), EStatus::ABORTED, result.GetIssues().ToString());        
+            runtime.SetObserverFunc(saveObserver);
```

Comment:
I would suggest to use to following construction instead

Y_DEFER {
   runtime.SetObserverFunc(saveObserver);
}

Reply:
by the way, thank you for fixing that.